### PR TITLE
[opinfo] tranpose_conv, conv, adaptive_{max, avg}_pool unbatched samples

### PR DESCRIPTION
**Pull request by kshitij12345**
As per title

### CODE REVIEW

**Comment by kshitij12345** [comment]
@samdow 

This PR tries to add unbatched sample for a few `nn.functional` operators

Following tests in TestExpandedWeights fail as they always assume that batch_size is present.
`test_expanded_weight_per_sample_grad_nn_functional_conv1d_cpu_float64`
`test_expanded_weight_per_sample_grad_nn_functional_conv2d_cpu_float64`

I checked the code and it seems `conv_expanded_weights` always assumes the first dim to be batch dim
https://github.com/pytorch/pytorch/blob/7cd8857b8ab46e4308d09788b0457cdc2c9d33a5/torch/nn/utils/_expanded_weights/conv_expanded_weights.py#L23

Also the backward assumes the same
https://github.com/pytorch/pytorch/blob/7cd8857b8ab46e4308d09788b0457cdc2c9d33a5/torch/nn/utils/_expanded_weights/conv_utils.py#L53-L62

I think we should just skip the unbatched samples in the test and open an issue for the same.

Wdyt?

Thanks!!

**Comment by samdow** [comment]
@kshitij12345 Thanks for bringing this up and identifying the issue so thoroughly! I totally agree with skipping and filing an issue. I'll update the ExpandedWeights exclusions after

**Comment by kshitij12345** [comment]
@jbschlosser CI is green. Please take a look :)

**Review by jbschlosser — APPROVED** [review]
LGTM!
